src/main/java/jpa/domain/Member.java

```diff
@@ -0,0 +1,50 @@
+package jpa.domain;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Member extends BaseTimeEntity {
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private Integer age;
```

Comment:
이 부분은 상황에 따라 다르겠지만, null이 들어가지 않도록 primitive 타입으로 정의하실수도 있습니다.

```suggestion
    private int age;
```